docs/upgrade.md

```diff
@@ -99,6 +99,13 @@ experimental_features:
   groups_enabled: false
 ```
 
+## Change in behaviour for PostgreSQL databases with unsafe locale
+
+Synapse now refuses to start when using PostgreSQL with incorrect values of `COLLATE` and
```

Comment:
With my pretend synapse admin hat on, it's not immediately clear what a correct value of `COLLATE` and `CTYPE` is. It's not mentioned in https://matrix-org.github.io/synapse/latest/postgres.html#fixing-incorrect-collate-or-ctype either. I'm guessing it's "C"?

Reply:
Yes, this change applies to any non-`C` locale